library/src/scala/quoted/Quotes.scala

```diff
@@ -3517,7 +3515,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         def pos: Position
 
         /** The documentation for this symbol, if any */
-        def documentation: Option[Documentation]
+        def docstring: Option[String]
```

Comment:
```suggestion
        def rawDocumentation: Option[String]
```

Reply:
`rawDocumentation` makes me expect that there's a way to access non-raw documentation, which is not the case. I specifically picked a name which wasn't used in the previous API, so that we avoid collisions when reintroducing it in the future. `docstring` is used in a bunch of languages to refer to a definition's documentation, so I think it makes sense here as well.

Reply:
ok